packages/form-js-viewer/src/render/context/FormRenderContext.js

```diff
@@ -1,6 +1,9 @@
 import { createContext } from 'preact';
 
 const FormRenderContext = createContext({
+  EmptyRoot: (props) => {
```

Comment:
So we used to have Empty, which was basically rendered inside all grids (which used to be only the root) when they are empty. This is useful for stuff like rendering the "drag in here" image. I split it into two different component, Empty and EmptyRoot, so that we don't render this inside every grid but just the Root one.

![grafik](https://github.com/bpmn-io/form-js/assets/17801113/b7039a36-ae9d-44e7-8115-c8ba0ac34037)

I've kept the Root as we might want to keep this open for external or internal customization in the future.